src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatDataBlock.tsx

```diff
@@ -0,0 +1,57 @@
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import useKeyStatQuery from '@common/modules/find-statistics/hooks/useKeyStatQuery';
+import React, { ReactNode } from 'react';
+import KeyStat from '@common/modules/find-statistics/components/KeyStat';
+
+export interface KeyStatDataBlockProps {
+  children?: ReactNode;
+  releaseId: string;
+
+  dataBlockId: string;
+  trend?: string;
+  guidanceTitle?: string;
+  guidanceText?: string;
+
+  testId?: string;
+}
+
+const KeyStatDataBlock = ({
+  children,
+  releaseId,
+  dataBlockId,
+  trend,
+  guidanceTitle = 'Help',
+  guidanceText,
+  testId = 'keyStat',
+}: KeyStatDataBlockProps) => {
+  const { value: dataBlockResults, isLoading, error } = useKeyStatQuery(
+    releaseId,
+    dataBlockId,
+  );
+
+  if (error) {
+    return null;
+  }
+
+  if (isLoading) {
+    return <LoadingSpinner />;
```

Comment:
Pretty sure we can LoadingSpinner can wrap the keystat component instead of having to be in its own `if` statement (similar to my other comment)